torch/_inductor/pattern_matcher.py

```diff
@@ -1642,7 +1642,7 @@ def compute_mutation_region_ids(graph: torch.fx.GraphModule) -> None:
 class PatternMatcherPass:
     def __init__(
         self,
-        prevent_match_across_mutations: bool = False,
+        prevent_match_across_mutations: bool = True,
```

Comment:
Should we just remove this flag entirely?

Reply:
Updated PR to remove this flag entirely